### ISSUE TITLE
[Feature Request] Add GitHub link to credential modal & dashboard

### ISSUE THREAD

**Issue by sdairs** [tinybirdco/log-analytics-starter-kit#9]
### Describe the feature you'd like to request

Can we add a GitHub link/button both to the Credentials modal & somewhere on the dashboard?

In the Credentials modal there should be a button to get to the GitHub repo, but also an area some text to send people to the README. Right now, if you land on the page & you don't already know what you're doing, you get stuck in this modal being asked for a token, but there's nothing saying what the token is, how to get it, why you should care, etc. The README for this repo is the best place to find that information atm, so we should try to guide a user to this repo ("Not sure what to do next? The README for this Starter Kit explains how to get started" or something like this)

On the Dashboard, we should have a button that takes the user to the repo. We will have a live demo running, so we want users to be able to easily find the repo from the live demo.

### Describe the solution you'd like

GitHub button

**Comment by joaqo** [comment]
Is there any way to populate the graphs to see how this would look? I created an account in tiny bird and a dashboard there too to be able to get past the log-in screen. But my charts are all empty. Pressed the "Generate data" button, but nothing happened.

**Comment by sdairs** [comment]
@joaqo seems there is a big with the generator embedded in the UI. 

You should be able to go to the Mockingbird UI (this is a UI for the data generator) https://mockingbird.tinybird.co/?schema=Log+Analytics+Starter+Kit&host=eu_gcp&token=&datasource=logs&eps=1 and generate some fake data for your charts. Make sure to hit the `Tinybird settings` in the top right, and add your Auth Token & select the correct region (EU/US).

I've raised an issue to fix the data generator embedded in the dashboard https://github.com/tinybirdco/log-analytics-starter-kit/issues/32

**Comment by joaqo** [comment]
Hey, thanks a lot for replying. I tried your suggestion and I am successfully ingesting data into my tiny bird account, as I can see it on the tiny bird dashboard. Still, for some reason this log analytics repo is not displaying them when I run it locally. I think I'm setting the token and everything else correctly, so not sure what the issue is.

**Comment by sdairs** [comment]
@joaqo can you try something for me - using the date picker, can you select a custom date range that covers 3 days, yesterday, today, and tomorrow (i.e. if today is the 12th, then pick 11, 12 & 13) and see if it displays todays data?

**Comment by joaqo** [comment]
I found the issue! Turns out I had selected the wrong server. I knew I had to select the same server I selected on my tiny bird account, but it wasn't clear that the Host menu on the initial screen was actually selecting Europe vs USA. This is what you're prompted with:

<img width="620" alt="image" src="https://user-images.githubusercontent.com/14966348/224586936-28b8f748-8c29-4417-9080-887ce4813902.png">

It does say us in the server name, but its different from how its presented on the tiny bird site (Europe vs USA), so maybe that could be something to improve.

Anyways, thanks for the great repo!
